packages/frontend/src/routes/LoginWrapper.js

```diff
@@ -28,12 +28,13 @@ const LoginWrapper = () => {
     const failureUrl = URLParams.failure_url;
     const successUrl = URLParams.success_url;
     const invalidContractId = URLParams.invalidContractId;
-
+    const methodNames = URLParams.methodNames;
     const contractIdUrl = `${CONFIG.EXPLORER_URL}/accounts/${contractId}`;
 
     const accountLocalStorageAccountId = useSelector(selectAccountLocalStorageAccountId);
 
-    let requestingFullAccess = !contractId || (publicKey && contractId?.endsWith(`.${CONFIG.LOCKUP_ACCOUNT_ID_SUFFIX}`)) || contractId === accountLocalStorageAccountId;
+    let requestingFullAccess = !methodNames || !contractId || (publicKey && contractId?.endsWith(`.${CONFIG.LOCKUP_ACCOUNT_ID_SUFFIX}`)) || contractId === accountLocalStorageAccountId;
```

Comment:
What they mean there is that if you don't specify `methodNames` then the key generated will have access to all methods of the specific contract (instead of just the ones specified in `methodNames`). That does not mean its a "full access" key for your wallet account.